src/apps/aave/positions.ts

```diff
@@ -65,26 +65,28 @@ const hook: PositionsHook = {
       args: [aaveAddresses.poolAddressesProvider],
     })
 
-    const [userReserveData, [reserveIncentiveData, _userIncentivesData]] =
-      address
-        ? await client.multicall({
-            contracts: [
-              {
-                address: aaveAddresses.uiPoolDataProvider,
-                abi: uiPoolDataProviderV3Abi,
-                functionName: 'getUserReservesData',
-                args: [aaveAddresses.poolAddressesProvider, address as Address],
-              },
-              {
-                address: aaveAddresses.uiIncentiveDataProvider,
-                abi: uiIncentiveDataProviderV3Abi,
-                functionName: 'getFullReservesIncentiveData',
-                args: [aaveAddresses.poolAddressesProvider, address as Address],
-              },
-            ],
-            allowFailure: false,
-          })
-        : [undefined, [undefined, undefined]]
+    const [
+      [userReserveData, _userEmodeCategoryId] = [],
```

Comment:
This is the only change to the multicall. The return type of getUserReservesData changed to a tuple in this latest contract